x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/state.ts

```diff
@@ -61,12 +61,13 @@ export interface State {
     destinationIndexNameEmpty: boolean;
     destinationIndexNameValid: boolean;
     destinationIndexPatternTitleExists: boolean;
-    earlyStoppingEnabled: undefined | boolean;
     downsampleFactor: undefined | number;
+    earlyStoppingEnabled: undefined | boolean;
     eta: undefined | number;
     etaGrowthRatePerTree: undefined | number;
     featureBagFraction: undefined | number;
     featureInfluenceThreshold: undefined | number;
+    featureProcessors: undefined | Array<Record<string, any>>;
```

Comment:
Nit: Any chance we can get rid of the `any`?

Reply:
Added `FeatureProcessor` type in c3c130b8e78556089a1005980328d7bd20b2664a